### PR TITLE
electron-bridge: Implement electron bridge

### PR DESCRIPTION
**Pull request by abhigyank**
This PR completes the minor work left from #428 .

### CODE REVIEW

**Comment by priyank-p** [comment]
@abhigyank Thanks for finishing it off! LGTM.

File: app/renderer/js/electron-bridge.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const events = require('events');
+const { ipcRenderer } = require('electron');
+
+// we have and will have some non camelcase stuff
+// while working with zulip so just turning the rule off
+// for the whole file.
+/* eslint-disable camelcase */
+class ElectronBridge extends events {
+	// eslint-disable-next-line no-useless-constructor
+	constructor() {
```
Comment (priyank-p) [inline app/renderer/js/electron-bridge.js]: Minor Thing, can you remove this thing, this is not required anymore.

**Review by priyank-p — APPROVED** [review]

File: app/renderer/js/electron-bridge.js (hunk 1 above)
Comment (abhigyank) [reply app/renderer/js/electron-bridge.js]: Thanks, done!

**Comment by akashnimare** [comment]
Merged, thanks, @priyankp10, @abhigyank. We now have a working bridge between the webapp and the desktop app :construction:

**Comment by timabbott** [comment]
So, this handles the icon changing immediately if the app was running when the admin changes it.

If the realm icon is changed while the app isn't running, do we handle that already?

**Comment by akashnimare** [comment]
> If the realm icon is changed while the app isn't running, do we handle that already?

Yeah, we do have a [function](https://github.com/zulip/zulip-electron/blob/781883a898ed30f2ddc95928cbacb945230ab5ed/app/renderer/js/utils/domain-util.js#L248) which lazy loads the realm icon but this needs a bit of work. Current flow is -
* Someone updated the realm icon while app isn't running
* User start the app they'll see the old icon
* On reload/re-starting the app again they'll see the new icon

**Comment by timabbott** [comment]
Cool.  If it were helpful for this sense, we could trigger an electron_bridge event on webapp startup; we'd just want to make sure the app is avoiding doing unnecessary work on every load if the icon URL hasn't changed.

**Pull request merged**